lib/worker.ex

```diff
@@ -28,9 +28,9 @@ defmodule GymTcpApi.Worker do
     send(caller, {:response, response, current})
 
     receive do
-      {:data, message, c} ->
+      {:data, message, _c } ->
           handle_call({message, caller}, :ok, python);
-      {:close, message} -> :python.call(python, :worker, :process_response, [""])
+      {:close, _message} -> :python.call(python, :worker, :process_response, [""])
```

Comment:
Is there any special reason to use underbar`_`?

Reply:
when I ran the server, it showed me a warning to prefix underscore for unused variables.

Reply:
Ah, always nice to remove some warnings.